components/sidebar_right/sidebar_right.jsx

```diff
@@ -140,7 +140,7 @@ export default class SidebarRight extends React.Component {
 
         var searchForm = null;
         if (this.props.currentUser) {
-            searchForm = <SearchBox isFocus={this.props.searchVisible && Utils.isMobile()}/>;
```

Comment:
I guess this will apply to the webapp as well? One thing this might break is the `/search` slash command, which leaves the focus in the message box.

Reply:
Yes.
I don't think it will break it. I just tested and the focus remains at message box after `/search` command.

Reply:
This will [set focus](https://github.com/mattermost/mattermost-webapp/blob/master/components/search_bar/search_bar.jsx#L266) to SearchBar if `isFocus` is true and `searchTerms` is empty.
In the case of `/search` command, search term is placed at SearchBar.